<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LAPTOP-JD0NA4B0" failures="0" tests="3" name="TestNG.TimeOut" time="3.016" errors="1" timestamp="2024-02-08T22:34:30 IST" skipped="0">
  <testcase classname="TestNG.TimeOut" name="TestA" time="0.003"/>
  <testcase classname="TestNG.TimeOut" name="TestB" time="3.007">
    <error message="Method TestNG.TimeOut.TestB() didn&#039;t finish within the time-out 3000" type="org.testng.internal.thread.ThreadTimeoutException">
      <![CDATA[org.testng.internal.thread.ThreadTimeoutException: Method TestNG.TimeOut.TestB() didn't finish within the time-out 3000
at java.base@18.0.2.1/java.lang.Thread.sleep(Native Method)
at app//TestNG.TimeOut.TestB(TimeOut.java:34)
at java.base@18.0.2.1/java.lang.invoke.LambdaForm$DMH/0x0000000800d14800.invokeVirtual(LambdaForm$DMH)
at java.base@18.0.2.1/java.lang.invoke.LambdaForm$MH/0x0000000800d18000.invoke(LambdaForm$MH)
at java.base@18.0.2.1/java.lang.invoke.Invokers$Holder.invokeExact_MT(Invokers$Holder)
at java.base@18.0.2.1/jdk.internal.reflect.DirectMethodHandleAccessor.invokeImpl(DirectMethodHandleAccessor.java:154)
at java.base@18.0.2.1/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:104)
at java.base@18.0.2.1/java.lang.reflect.Method.invoke(Method.java:577)
at app//org.testng.internal.invokers.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:139)
at app//org.testng.internal.invokers.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:47)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:76)
at app//org.testng.internal.invokers.InvokeMethodRunnable.call(InvokeMethodRunnable.java:11)
at java.base@18.0.2.1/java.util.concurrent.FutureTask.run(FutureTask.java:264)
at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1136)
at java.base@18.0.2.1/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
at java.base@18.0.2.1/java.lang.Thread.run(Thread.java:833)
]]>
    </error>
  </testcase> <!-- TestB -->
  <testcase classname="TestNG.TimeOut" name="TestC" time="0.006"/>
</testsuite> <!-- TestNG.TimeOut -->
